XCast/XCast.json

```diff
@@ -94,7 +94,7 @@
             }
         },
         "getFriendlyName":{
-            "summary": "Gets the human readable name for the implementation",
+            "summary": "Gets the human readable name for the implementation.\n  \n### Events \n\n No Events",
```

Comment:
Returns the friendly name set by setFriendlyName API.

Reply:
Updated changes as per your review comments